features/permissions/permissions.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Permissions
```

Comment:
Do we want any negative test cases? Like passing in a bad id and seeing the error come back?